tsconfig.json

```diff
@@ -6,19 +6,21 @@
     "lib": ["ES2022", "DOM", "DOM.Iterable"],
     "skipLibCheck": true,
     "jsx": "react-jsxdev",
-
     /* Bundler mode */
     "moduleResolution": "bundler",
     "allowImportingTsExtensions": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-
     /* Linting */
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    /* Aliases */
+    "paths": {
+      "~/*": ["./src/ui/*"]
```

Comment:
Curious about making ~ -> src/ui instead of src.

Also might want to add this to prettier `"importOrder"` in package.json

Reply:
> Curious about making ~ -> src/ui instead of src.

I usually do this because you're (almost) never going to import from `src` without importing from `src/ui`

> Also might want to add this to prettier `"importOrder"` in package.json

👍 done